server/api/db/sqldb/db.py

```diff
@@ -1797,9 +1810,10 @@ def _get_function_uid(
                 )
             return tag_function_uid
 
-    def _delete_functions(self, session: Session, project: str):
-        for function_name in self._list_project_function_names(session, project):
-            self.delete_function(session, project, function_name)
+    def _delete_project_functions(self, session: Session, project: str):
+        self.delete_functions(
+            session, project, self._list_project_function_names(session, project)
+        )
```

Comment:
I prefer if you do 
```
function_names = self._list_project_function_names(session, project)
```
Because it will be easier to read on stacktrace if it fails